docs/desktop-wpf/index.yml

```
@@ -0,0 +1,65 @@
+### YamlMime:Landing
+
+title: Windows Presentation Foundation (WPF) documentation
+summary: Learn about Windows Presentation Foundation, an open-source, graphical user interface for Windows.
+
+metadata:
+  title: Windows Presentation Foundation (WPF) for .NET Core documentation
+  description: Learn about Windows Presentation Foundation, an open-source, graphical user interface for Windows.
+  ms.topic: landing-page
+  ms.date: 03/25/2020
+
+# linkListType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | sample | tutorial | video | whats-new
+
+landingContent:
+
+  # Card
+  - title: Learn about WPF
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Windows Presentation Foundation overview
+            url: overview/index.md
+          - text: XAML in Windows Presentation Foundation
+            url: fundamentals/xaml.md
+
+  # Card
+  - title: Get started with WPF
+    linkLists:
+      - linkListType: overview
```

These articles don't sound like overview to me.

```suggestion
      - linkListType: concept
```